packages/sdk/src/sdk/services/Storage/Storage.ts

```diff
@@ -121,9 +121,22 @@ export class Storage implements StorageService {
     Object.keys(options).forEach((key) => {
       formData.append(key, `${options[key]}`)
     })
+
+    const formDataFile =
+      'uri' in file
+        ? {
+            ...file,
+            // NOTE this is required for react-native
+            // certain characters in the file name make formData invalid
+            name: file.name
+              ? encodeURIComponent(file.name.replace(/[()]/g, ''))
```

Comment:
Wild! Do parentheses not get encoded w/ encodeURIComponent I guess?

Reply:
Nope :(
And encodeURIComponent wasn't really even necessary here but I wanted to try and rule out any other problem characters